api/src/routes/api.ts

```diff
@@ -1,10 +1,40 @@
+import express = require("express");
+
 import CategoryCollection from "../models/CategoryCollection";
 import Category from "../models/Category";
 import Config from "../models/Config";
+import { getUserBy, makeToken } from "../services/Database";
 import Job from "../models/Job";
 
-import express = require("express");
 const router = express.Router();
+import { /* allow, */ setupPassport, requireAuth, requireToken } from "./auth";
```

Comment:
Why is allow commented out here? Is this something we're going to add in future, or is it no longer needed?